docs/user-guide/vpcs.md

```diff
@@ -12,8 +12,8 @@ metadata:
   name: vpc-1
   namespace: default
 spec:
-  ipv4Namespace: default # Limits to which subnets could be used by VPC to guarantee non-overlapping IPv4 ranges
-  vlanNamespace: default # Limits to which switches VPC could be attached to guarantee non-overlapping VLANs
+  ipv4Namespace: default # Limits what subnets can the VPC use to guarantee non-overlapping IPv4 ranges
+  vlanNamespace: default # Limits what Vlan Ids can the VPC use to guarantee non-overlapping VLANs
```

Comment:
Limits which Vlan Ids the VPC can use in order to guarantee non-overlapping VLANs

...I think